user/id.go

```diff
@@ -67,6 +75,20 @@ func (x *ID) SetScriptHash(scriptHash util.Uint160) {
 	copy(x.w[21:], hash.Checksum(x.w[:21]))
 }
 
+// GetScriptHash gets scripthash from user ID.
+func (x *ID) GetScriptHash() (util.Uint160, error) {
```

Comment:
i dont think such signature would be convenient to use. How about
```
// GetScriptHash gets script hash from user ID.
func (x ID) GetScriptHash() util.Uint160 {
	return util.Uint160(x.w[1:21])
}
```
if `ID` constructed correctly - ok, otherwise panic, so be it